docs/operator-docs/run-a-stress-test.md

```diff
@@ -0,0 +1,51 @@
+---
+id: run-a-stress-test
+title: Run A Stress Test
+hide_title: true
+---
+
+# Run A Stress Test
```

Comment:
This stays in the wiki since it's so EED-specific. CloudTamer is an EED thing. `CNMTestHarness` might be specific to PODAAC. This might be something we need to figure out if they can share or just leave it out.

Reply:
But this is a section I do want (in the wiki) and want to work on. I think this could be super valuable.

Reply:
Removed this and put back in the wiki. Updated it there.